test_version/version_specific_tests_dir/twitter_scrooge/twitter_scrooge_bindings.bzl

```diff
@@ -0,0 +1,37 @@
+load("@io_bazel_rules_scala//twitter_scrooge:twitter_scrooge.bzl", "defaulted_twitter_scrooge_dependency", "twitter_scrooge")
+load("@io_bazel_rules_scala//scala:scala_cross_version.bzl", "extract_major_version")
+
+def twitter_scrooge_with_custom_dep_version(twitter_scrooge_deps_version, scala_version, version_shas):
+    scala_major_version = extract_major_version(scala_version)
+
+    scrooge_core_label = defaulted_twitter_scrooge_dependency(
```

Comment:
why do you use `defaulted_twitter_scrooge_dependency`? this doesn't demonstrate using a different loader and maybe even doesn't protect us. Maybe if I'll just change the bind to keep using `actual = "@io_bazel_rules_scala_scrooge_core` then it will pass because the test uses this convention as well

Reply:
That's fair.

Our use case is to test this with a source-level target, i.e. pointing these labels to the actual source code. Ideally, I'd reproduce that here by fetching scrooge-core via `http_archive` or `git_repository` with a custom label. However, public scrooge-core is not yet a Bazel project (but out internal version is, somewhat). Therefore, would it be okay for me to use plain `scala_maven_import_external` to fetch the appropriate version of the dependency with a custom label?

Reply:
yes, definitely

Reply:
just make sure to add a comment on the `scala_maven_import_external` saying it's for a test and maybe which test

Reply:
Got it, thanks!